lib/created/modalLocationRadiusCreated.js

```diff
@@ -1,90 +1,126 @@
-                                        scaledSize: new google.maps.Size(25, 25)
-                                    };
+                    var marker = new google.maps.Marker({
+                        map: map.instance,
+                        icon: image,
+                        title: place.name,
+                        draggable: true,
+                        position: place.geometry.location,
+                        animation: google.maps.Animation.DROP
+                    });
 
+                    // Create a marker for each place.
 
-                            var areaCircle = new google.maps.Circle({
-                                map: map.instance,
-                                center: place.geometry.location,
-                                zoom: 7,
-                                radius: 8093.4,
-                                strokeColor: "#f8504b",
-                                strokeOpacity: 0.8,
-                                strokeWeight: 2,
-                                fillColor: "#f8504b",
-                                fillOpacity: 0.4
-                            });
+                    markers.push(marker);
+                    console.log(marker);
 
-                            var marker = new google.maps.Marker({
-                                map: map.instance,
-                                icon: image,
-                                title: place.name,
-                                draggable: true,
-                                position: place.geometry.location,
-                                animation: google.maps.Animation.DROP
-                            });
 
-                                    // Create a marker for each place.
+                    var userlat = marker.getPosition().lat();
+                    var userlng = marker.getPosition().lng();
 
-                                    markers.push(marker);
-                                    console.log(marker);
+                    var locate = marker.getPosition();
+                    console.log(locate);
 
 
-                                    var userlat = marker.getPosition().lat();
-                                    var userlng = marker.getPosition().lng();
+                    Session.set("userlat", userlat);
+                    Session.set("userlng", userlng);
 
-                                    var locate = marker.getPosition();
-                                    console.log(locate);
+                    console.log(userlat, userlng);
+                    var latLng = userlat.toString() + ", " + userlng.toString();
 
+                    var mape = map.instance;
+                    mape.setCenter(marker.getPosition());
 
-                                    Session.set("userlat", userlat);
-                                    Session.set("userlng", userlng);
+                    function geocodeLatLng(geocoder, map, information) {
```

Comment:
'geocodeLatLng' is defined but never used.<br>'information' is defined but never used.<br>'map' is defined but never used.